app/helpers/hera_helper.rb

```diff
@@ -68,6 +71,19 @@ def navbar_brand
     end
   end
 
+  def node_types_collection
+    Node::Types::LABELS.map.with_index do |type, index|
```

Comment:
```suggestion
    Node::Types::LABELS.filter_map.with_index do |type, index|
```

If we use `filter_map` we don't need to call `compact` in line 84